src/main/java/com/minecolonies/coremod/entity/ai/citizen/cook/EntityAIWorkCook.java

```diff
@@ -229,6 +231,7 @@ else if (InventoryUtils.hasItemInItemHandler(handler, stack -> CAN_EAT.test(stac
             removeFromQueue();
             return getState();
         }
+        worker.getCitizenColonyHandler().getColony().getStatisticsManager().increment(FOOD_SERVED);
```

Comment:
Meanwhile this only counts by one "serving event", not the number of food actually served.  Which might be unexpected.

Reply:
Yeah, I thought a bunch about this, but there is no "nondisruptive" way to count this else (without significantly reworking the existing code). So it's "Food served"

Reply:
Could change `transferFoodUpToSaturation` to return number of items served, with 0 being equivalent to the current false return.  Or return the `foundSaturation` directly and have the stat be "food shanks gained from eating" or shorter words to that effect.